### PR TITLE
Send back user info on login

### LINKED ISSUE #337: /login should return basic User details
Such as username and email so that they can be stored in the global `App.session` object and retrieved for the navbar

### PR DESCRIPTION
**Pull request by Ommy**
Includes if user is super user to use to determine if they can upload
a concept dictionary CSV.

Resolves #337

### CODE REVIEW

File: src-django/authentication/views.py [hunk 1]
```diff
@@ -114,6 +115,14 @@ def login(request):
 
         token_key = token.key
         logger.info("Login Success: u:{0} e:{0}".format(user.username, user.email))
+
+        user_details = {
+            'super_user': user.is_superuser,
```
Comment (Trinovantes) [inline src-django/authentication/views.py]: keep the keys the same

File: src-django/authentication/views.py [hunk 2]
```diff
@@ -121,6 +130,7 @@ def login(request):
         'success': valid_form,
         'errors': form.errors,
         'token': token_key,
+        'user': user_details if user_details else False,
```
Comment (Trinovantes) [inline src-django/authentication/views.py]: this should return an empty object

File: src-django/authentication/views.py [hunk 3]
```diff
@@ -99,6 +99,7 @@ def login(request):
     form = AuthenticationForm(data=request.POST)
     valid_form = form.is_valid()
     token_key = None
+    user_details = None
```
Comment (Trinovantes) [inline src-django/authentication/views.py]: you could init this to `{}` so the return statement doesn't need to check for None

**Comment by Trinovantes** [comment]
+1

**Comment by Kapin** [comment]
:+1:

**Pull request merged**